.circleci/config.yml

```diff
@@ -5,18 +5,18 @@ orbs:
 workflows:
   package-and-push-chart-on-tag:
     jobs:
-    - architect/push-to-app-catalog:
-        context: architect
-          # executor: "app-build-suite" # uncomment this if you want automatic metadata generation and helm chart linting
-        name: package and push teleport-kube-agent-app chart
-        app_catalog: default-catalog
-        app_catalog_test: default-test-catalog
-        chart: teleport-kube-agent
+      - architect/push-to-app-catalog:
+          context: architect
+          executor: app-build-suite
+          name: package and push teleport-kube-agent-app chart
+          app_catalog: default-catalog
```

Comment:
I'm not entirely sure, but I believe you need to match the .abs config with this catalog (in other words, use default catalog as your `catalog-base-url`). IIRC there could be some breakage around upgrade testing if they don't match